### PR TITLE
fix(authoring topbar): Improved topbar in authoring

### PR DESCRIPTION
**Pull request by pavlovicnemanja92**
- [x] Removed status label [SD-2726],
- [x] Placed "Add to highlights" function inside 3 dots menu [SD-2827]
- [x] Improved responsive behavior [SD-2843].

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html [hunk 1]
```diff
@@ -87,9 +78,22 @@
                     </li>
                 </ul>
 
-                <ul ng-if="_editable"><li class="divider"></li></ul>
+                <ul ng-if="item.task && item.task.desk && itemActions.mark_item"
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: `item.task` is redundant, Angular already gracefully handles cases like `item.<somethingUndefined>.desk`, thusno need to check if `item.task` is a truthy value.

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html [hunk 2]
```diff
@@ -87,9 +78,22 @@
...
+                     title="{{ :: 'Mark item' | translate }}">
+                    <li class="divider"></li>
+                    <li><span class="menu-label">Highlights</span></li>
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: translate?

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html [hunk 3]
```diff
@@ -87,9 +78,22 @@
...
+                    <li>
+                        <div class="dropdown highlights-dropdown" dropdown>
+                            <button class="dropdown-toggle" dropdown-toggle>
+                                Mark item
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: translate?

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html (hunk 1 above)
Comment (pavlovicnemanja92) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: So, is it ok just to write `item && itemActions.mark_item` ?

Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: You probably mean `item.task.desk && itemActions.mark_item` ?
Yes, it is, Angular gracefully handles the cases when `item.task` is undefined.

Comment (pavlovicnemanja92) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: :+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by Aca-jov** [comment]
:+1:

**Pull request merged**